include/boost/url/impl/const_string.hpp

```diff
@@ -11,19 +11,19 @@
 #define BOOST_URL_IMPL_STRING_HPP
 
 #include <boost/url/detail/over_allocator.hpp>
+#include <atomic>
 
 namespace boost {
 namespace urls {
 
-struct string_value::base
+struct const_string::base
 {
-    std::size_t refs = 1;
+    std::atomic<std::size_t> refs{1};
     virtual void destroy() noexcept = 0;
 };
 
 template<class Allocator>
-auto
-string_value::
+auto const_string::
```

Comment:
The formatting changed everywhere

Reply:
Yes. The IDE tried its best when refactoring the class name. I'll go through the project changing it.

Reply:
You let the IDE do the rename?

Reply:
Partially, yes. It helped me locate the strings.

I went through every change in the project. It should be OK now.